lib/rules/no-unsafe.js

```diff
@@ -31,16 +31,29 @@ module.exports = {
       return {};
     }
 
+    const unsafe = {};
+    unsafe.componentWillMount = unsafe.UNSAFE_componentWillMount = {
```

Comment:
instead of making an object and mutating it, let's create it all in one literal? the various objects can be stored in variables and assigned like `{ componentWillMount, UNSAFE_componentWillMount: componentWillMount }` etc.

Reply:
fixed